_posts/2015-10-27-drupal-web-mapping.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: post
+title: "Drupalized Web Mapping @ NACIS 2015"
+date: 2015-10-27
```

Comment:
Before merging don't forget to set this to today's date. Might as well update the filename too if you're feeling up for it!